Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Interactable.cs

```diff
@@ -585,9 +644,17 @@ public void OnPointerUp(MixedRealityPointerEventData eventData)
                 return;
             }
 
-            if (ShouldListen(eventData.MixedRealityInputAction))
+            bool isGrab = IsGrab(eventData.Pointer, eventData.MixedRealityInputAction);
+
+            if (ShouldListen(eventData.MixedRealityInputAction) || isGrab)
```

Comment:
why not just put isGrab logic into ShouldListen? Why is isGrab special?

Reply:
This could have been a remnant of testing a keeping it separate until everything worked.

Reply:
Oh, wait. I'm using the IsGrab value to set the grab state. I'm also using it for handling the event. I could create an out value for grab then combine them.